src/validateSchema.js

```diff
@@ -8,20 +8,30 @@ const getValidationErrors = (schema, model) => Object.keys(schema).reduce((acc,
   const value = model[key]
   const rules = schema[key]
 
+  const renderError = (condition, fallback) => {
+    let err;
+    try {
+      err = rules.formatError({ key, value, condition, rules, schema, model });
+    } catch (e) {
+      err = rules.formatError || fallback;
```

Comment:
Fair enough, I can push those changes to this PR now to make it simpler for you. Sticking with a function does make it cleaner and more consistent 👍